### DISCUSSION TITLE
Return to Palace interior?

### DISCUSSION THREAD

**Discussion by LuminarLight** [open-goal/jak-project#2881] [Jak 2]
### Is your feature request related to a problem?

In the game called 'Jak II', you can never return to the Palace interior after its corresponding mission. This is a problem, because it is a cool area that I always wanted to revisit as a child.
The level had orbs in earlier versions, but in final version it has no orbs - so it doesn't contribute to the "missable orbs" problem of Jak II, fortunately.

### Describe the solution you'd like.

If you return to the level using cheats or debug menu, it is fully playable and stable and normal.

Once you complete the game, you are able to return to Fortress, and go through warp gates to revisit areas like the Strip Mine or Drill Platform.

But even after you complete the game, if you go to the palace main entrance, you will never be able to re-enter the Palace interior... the elevator won't be there.

What I suggest is to allow re-entering the palace interior using the main entrance's elevator, AFTER the game has been completed (to stay consistent with the ability to return to fortress and things like that).

### Additional context

I saw the deviation done for the Metal Head Nest, so I believe that this may get accepted as well.

If return to palace interior is made possible, there will be no more canonically non-ruined areas left in Jak II OpenGOAL that can not be revisited.
(Krew boss area is canonically ruined after the fight due to the explosion, but you can return there in final game without cheats - you will just not be able to ever leave.)

**Comment by ManDude** [comment]
Unless somebody else does it, it's highly unlikely we'll make returning to the palace interior work. The elevator to it conflicts with the palace roof and the doors and loading regions are only set up properly for going from the elevator directly to the throne room, which is also locked.

**Comment by LuminarLight** [comment]
It could be a fun challenge for me.
Indeed it may not be easy, and it is understandable that the main developers don't have motivation/time to work on it.
But at least the idea itself is not rejected.

I will see if I can manage something...

**Comment by LuminarLight** [comment]
#2789 opened a PR, draft for now
